### PR TITLE
[Docs] Add doc for condition flow with activate

### PR DESCRIPTION
**Pull request by PeiwenGaoMS**
# Description

- cd `./scripts/docs`
- run  `.\doc_generation.ps1 -WithReferenceDoc -WarningAsError:$true`
- check `./_build/how-to-guides/develop-a-flow/develop-conditional-flow.html`

## Document screenshot

image
image
image
image
image

# All Promptflow Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [ ] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](../CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### Testing Guidelines
- [x] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Review by weedqian — DISMISSED** [review]

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
+# Develop conditional flow
```
Comment (ttthree) [inline docs/how-to-guides/develop-a-flow/develop-conditional-flow.md:1]: @PeiwenGaoMS this is not a good place to put the doc.
The problem is that this adds confusion "conditional flow" is a parallel concept with "standard flow". It's not.

I would rename the doc to be "Add conditional control to your flow" and put it under the root folder of "how-to-guides".

On a side note - @wangchao1230 why don't we have develop-chat-flow and develop-evaluation-flow under the folder "develop-a-flow"?

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+In Prompt flow, we support control logic by activate config, like if-else, switch. Activate config enables conditional execution of nodes within your flow, ensuring that specific actions are taken only when the specified conditions are met.
+
+This guide will help you learn how to develop a conditional flow using activate config.
+
+## Prerequisites
+
+Please ensure that your promptflow version is greater than `0.1.0b5`.
+
+## Usage Description
```
Comment (ttthree) [inline docs/how-to-guides/develop-a-flow/develop-conditional-flow.md:15]: "Usage" :)

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+![visual_editor](../../media/how-to-guides/conditional-flow-with-activate/visual_editor.png)
+
+- Click on the `Activation config` section in the node you want to add and fill in the values for "when" and "is".
+![activate_config](../../media/how-to-guides/conditional-flow-with-activate/activate_config.png)
+
+:::
+
+::::
+
+### Notice
```
Comment (ttthree) [inline docs/how-to-guides/develop-a-flow/develop-conditional-flow.md:49]: Something like "Further details and important notes" would work better.
And i would probably order the 3 items as #2, #3, #1 to emphasize these parts that can lead to issue.

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+1. There are two scenarios where nodes might be bypassed even without an activate config or when conditions aren't met:
```
Comment (ttthree) [inline docs/how-to-guides/develop-a-flow/develop-conditional-flow.md:50]: 1. Add an actual screenshot and explain what is "bypassed".
2. "A step/node can be bypassed with following scenarios: 1), 2), ..." and #1 is obviously activate.when is evaluated as false. For completeness we shouldn't assume user knows this.

File: docs/how-to-guides/quick-start.md [hunk 5]
```diff
@@ -284,7 +284,7 @@ See more details of this topic in [Initialize and test a flow](./init-and-test-a
 ## Next steps
 
 Learn more on how to:
-- [Develop a flow](./develop-a-flow/index.md): details on how to develop a flow by writing a flow yaml from scratch.
+- [Develop a flow](./develop-a-flow/index.md): details on how to develop a flow by writing a flow yaml from scratch and how to develop a conditional flow.
```
Comment (ttthree) [inline docs/how-to-guides/quick-start.md:287]: As commented above, decouple "develop conditional flow" from "develop a flow"

**Review by ttthree — CHANGES_REQUESTED** [review]

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md (hunk 1 above)
Comment (wangchao1230) [reply docs/how-to-guides/develop-a-flow/develop-conditional-flow.md:1]: we have the develop-chat-flow in this PR: https://github.com/microsoft/promptflow/pull/554
for develop evaluation flow, I created a task on this.
[Task 2703094](https://msdata.visualstudio.com/Vienna/_workitems/edit/2703094): Add doc: how to develop evaluate flow

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md (hunk 2 above)
Comment (PeiwenGaoMS) [reply docs/how-to-guides/develop-a-flow/develop-conditional-flow.md]: Updated.

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md (hunk 4 above)
Comment (PeiwenGaoMS) [reply docs/how-to-guides/develop-a-flow/develop-conditional-flow.md:50]: Updated.

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md (hunk 3 above)
Comment (PeiwenGaoMS) [reply docs/how-to-guides/develop-a-flow/develop-conditional-flow.md]: Updated.

File: docs/how-to-guides/quick-start.md (hunk 5 above)
Comment (PeiwenGaoMS) [reply docs/how-to-guides/quick-start.md:287]: Updated.

File: docs/how-to-guides/develop-a-flow/develop-conditional-flow.md (hunk 1 above)
Comment (PeiwenGaoMS) [reply docs/how-to-guides/develop-a-flow/develop-conditional-flow.md]: Updated.

**Review by thy09 — APPROVED** [review]

**Review by Hhhilulu — APPROVED** [review]

**Pull request merged**